administrative-sdk/connection.js

```diff
@@ -229,46 +232,44 @@ class Connection {
    * @param {callback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.